plugins/calendar/pkg/calendarImp.go

```diff
@@ -0,0 +1,140 @@
+package pkg
+
+import (
+	"bytes"
+	"encoding/csv"
+	"errors"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+	"time"
+)
+
+const (
+	dbFile         = "calendar_db.csv"
```

Comment:
Those variables should not be hardcoded in this file since the package can be used multiples times, would be a nice to accept these configurations as parameters on the New() func.

``` 
func New(file, collumns, title string, fileMode integer) &Calendar {
    return &Calendar{
             file:file,
             collumns:collumns,
             ... 
    } 
}
```